CoreService/Workflow.psm1

```diff
@@ -0,0 +1,231 @@
+* Private members
+**************************************************
+#>
+
+<#
+**************************************************
+* Public members
+**************************************************
+#>
+
+function Get-TridionWorkflowItems
+{
+    <#
+    .Synopsis
+    Gets a list of Workflow Items in Tridion Content Manager By Status or AssignedTo Filter
+    .Description
+    Gets a list of Workflow Items in Tridion Content Manager By Status or AssignedTo Filter
+    .Notes
+    Example of properties available: Id, Title Etc.
+    
+    For a full list, consult the Content Manager Core Service API Reference Guide documentation 
+    (Tridion.ContentManager.Data.CommunicationManagement.ProcessInstanceData object)
+    .Inputs
+     [string] Status(Not Mandetory): Workflow ProcessInstanceData by status "Active" or "Hstoricle" (Bydefault is will show all the data)
+	 [string] AssignedToById: The processInstanceData assigned to the given ID of User or Group.
+     [string] AssignedToByTitle: The processInstanceData assigned to the given Title of User or Group. (For User provide : <domainname>\<username>)
+	
+    .Outputs
+    Returns a list of objects of type [Tridion.ContentManager.CoreService.Client.ProcessInstanceData].
+	
+    .Link
+    Get the latest version of this script from the following URL:
+    https://github.com/pkjaer/tridion-powershell-modules
+	
+    .Example
+    Get-TridionWorkflowItems
+	Returns a list of all workflow items within Tridion (Active and Historical).
+	
+	.Example
+    Get-TridionWorkflowItems -Status "Active"
+	Returns a list of all workflow items within Tridion filtered By Status(Active) of The Workflow.
+	
+	.Example
+	Get-TridionWorkflowItems -Status "Historical"
+	Returns a list of all workflow items within Tridion filtered By Status(Historical) of The Workflow.
+	
+	.Example
+    Get-TridionWorkflowItems -AssignedToById "tcm:0-1014-65552"
+	Get-TridionWorkflowItems -AssignedToById "tcm:0-15-65568"
+	Returns a list of all workflow items within Tridion filtered by AssignedTo given ID of User or Group.
+	
+	.Example
+    Get-TridionWorkflowItems -AssignedToByTitle "TRIDIONDEV\San"
+	Get-TridionWorkflowItems -AssignedToByTitle "Developer"
+	Returns a list of all workflow items within Tridion filtered by AssignedTo given Title of User or Group.
+	
+	.Example
+    Get-TridionWorkflowItems -Status "Active" | Select-Object Title, Id
+	Get-TridionWorkflowItems -AssignedToById "tcm:0-1014-65552" | Select-Object Id,Title
+	Returns a list of the Title, Id of all workflow items within Tridion filtered By Status(Active) or AssignedTo filter of The Workflow.
+    #>
+    [CmdletBinding(DefaultParameterSetName='Status')]
+    Param
+    (
+		[Parameter(ParameterSetName='Status')]
+		[ValidateSet('', 'Active', 'Historical')]
+		[string]$Status,
+		
+		# The TCM URI of the Publication Target to load.
+        [Parameter(ParameterSetName='AssignedToById')]
+		[ValidateNotNullOrEmpty()]
+        [string]$AssignedToById,
```

Comment:
Maybe we could combine these into one 'AssignedTo' parameter? I think that would be easier to use. 
We could just check if it's a TCM URI (=by ID) or not (=by title).